app/javascript/src/components/Dashboard/Notes/utils.js

```diff
@@ -0,0 +1,9 @@
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+dayjs.extend(relativeTime);
```

Comment:
- You have to pay attention to the Slack thread. 
- More than that you have to carefully go through the React Best Practices book. We took effort to write that material so that we don't have to repeat the same points like I am doing right now. Read: https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=How%20to%20extend%20day.js%3F

Reply:
Extremely sorry for this mistake. I will be fixing this in my next PR.